pkg/manager/manager_test.go

```diff
@@ -64,6 +66,48 @@ func (l *FakeLink) Type() string {
 	return "FakeLink"
 }
 
+type FakeBondLink struct {
+	netlink.LinkAttrs
+}
+
+func (l *FakeBondLink) Attrs() *netlink.LinkAttrs {
+	return &l.LinkAttrs
+}
+
+func (l *FakeBondLink) Type() string {
+	return "bond"
+}
+
+var _ = Describe("IPCLock", func() {
```

Comment:
i can find several issues with the tests below

- first test will not test new path creation from second test invocation 
- third test will work (fail) if executed as root and will leave residual files in the system

What i propose is either:
1. use a `BeforeAll` block and use `tempDir = GinkgoT().TempDir()`  to create a temp dir which will get cleaned up if all tests pass. in this dir you can create folder with/without permissions for the individual tests
2. move pkg/cache/fs_ops.go to its own pacakge and use its interface in Lock implementation. then you can mock file system operation behaviour